packages/fortress/data/index.js

```diff
@@ -0,0 +1,72 @@
+const data = {
+  global: {
+    title: 'Firefox Fortress',
+    tagline: 'Achieve your fortress dreams',
+    planPicker: 'Select your plan',
+    congrats: 'You are a hero of time and space',
+    paymentsUrl: '//localhost:3030/subscriptions/products',
```

Comment:
This is fine so long as this doesn't get deployed anywhere which, I don't believe it does?

Reply:
oh yeah, i was thinking about doing a follow up where i put the fortress images on our CDN...